odoo/addons/base/models/ir_qweb.py

```diff
@@ -430,7 +430,7 @@
 RSTRIP_REGEXP = re.compile(r'\n[ \t]*$')
 VARNAME_REGEXP = re.compile(r'^[A-Za-z_][A-Za-z0-9_]*$')
 # Attribute name used outside the context of the QWeb.
-SPECIAL_DIRECTIVES = {'t-translation', 't-ignore', 't-title'}
+SPECIAL_DIRECTIVES = {'t-translation', 't-ignore', 't-title', 't-forbid-sanitize'}
```

Comment:
Sorry for not seeing your PM today, but yes ideally to be added in web_editor, not in base.

To be moved to a class method instead of a constant? as it is already done with https://github.com/odoo/odoo/blob/6922a40d53528524f122bff49255d0117a2b2739/odoo/addons/base/models/ir_qweb.py#L572
Probably also to be inherited by other modules

Reply:
@rdeodoo You replied at the same minute as CHM ^^ (I guess after CHM changes, this is to be ignored :+1: )